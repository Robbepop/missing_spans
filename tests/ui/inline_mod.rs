#![feature(proc_macro_hygiene)]

use missing_spans::my_attribute;

#[my_attribute]
mod foo {}

fn main() {}
