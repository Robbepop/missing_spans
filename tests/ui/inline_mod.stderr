error: attribute on module
 --> $DIR/inline_mod.rs:6:5
  |
6 | mod foo {}
  |     ^^^
